README.md

```diff
@@ -29,6 +29,37 @@ For example:
 This will create a PrometheusRule based on the `5xx-rate.tmpl` template with a
 threshold of `0.001`.
 
+## Deployment Annotations
+
+Your Deployment must have annotations in the form of:
+
+`com.uswitch.heimdall/<prometheus-rule-name>: <threshold>`
+
+For example:
+
+`com.uswitch.heimdall/5xx-rate-deployment: "0.001"`
+
+This will create a PrometheusRule based on the `5xx-rate-deployment.tmpl` template with a
+threshold of `0.001`.
+
+Available annotations:
+
+`com.uswitch.heimdall/4xx-rate-deployment` - alerts if the 4XX rate goes above the given threshold for at least 1 minute
+`com.uswitch.heimdall/5xx-rate-deployment` - alerts if the 5XX rate goes above the given threshold for at least 1 minute
+`com.uswitch.heimdall/p95-deployment` - alerts if the P95 goes above the given ms for at least 5 minutes
+`com.uswitch.heimdall/p99-deployment` - alerts if the P95 goes above the given ms for at least 5 minutes
+`com.uswitch.heimdall/replicas-availability-deployment`- alerts if the given % of replicas are not running for 5 minutes
```

Comment:
you can create whatever you like, these are ones we've created.

Reply:
Apologies for this, but can you please elaborate on this statement? I am not sure if I get that right. Many thanks :+1:

Reply:
These are completely arbitrary, your statement makes it seem like they're the only ones available. A user can create whatever templates and therefore annotations that they desire

Reply:
I don't necessarily agree, all it says, we currently have this set of automated alerting available. 
Of course, no one is limited to use only these, but in order to create new templates, they might need to tweak the coding part to ensure they have (for example) the Prometheus Alert pointed to the right Prometheus instance.

If you find it would be useful, I am happy to attach a section mentioning that though these are the SRE-created and tested alerts, but we also allow people to come up with ideas and create PRs for additional Prometheus Rules if they feel the need.

Reply:
This is an open source repo, the rules in here are meant to be examples that anyone can use, Heimdall is meant to just generate rules from templates + annotations. If someone deploys heimdall they can define what rules they like, they may now want to use the ones we have in the examples.

Reply:
So in the current version available already has a section explaining how templates can be overridden and customised.
This new section I added is a duplicate of the `Ingress` examples.
Would you like me to add some further clarification to the existing section above to clarify this point? It might be actually a good idea, now that we have multiple options to choose from. Is that, what makes it look more confusing?

Reply:
I think we should just make it clear that we have examples in this repo for these things, the way it's currently phrased makes it sound like they're the `only` things that are `available`. It's also probably worth stressing that these examples are somewhat specific to your environment and may not work without changing them significantly.

Reply:
I find that a fair point. I will make sure to update the README accordingly, to make this much clearer. Thanks a lot for the feedback! :+1: